lib/nerves_network/application.ex

```diff
@@ -9,14 +9,19 @@ defmodule Nerves.Network.Application do
 
     [resolvconf_file: resolvconf_file] = Application.get_env(:nerves_network, :resolver, [])
     [ipv4: ipv4] = Application.get_env(:nerves_network, :dhclientv4, [])
+    [ipv6: ipv6] = Application.get_env(:nerves_network, :dhclientv6, [])
 
-    dhclientv4_config_file = ipv4[:config_file] || Nerves.Network.Dhclientv4Conf.default_dhclient_conf_path()
+    dhclientv4_config_file = ipv4[:config_file] || Nerves.Network.DhclientConf.default_dhclient_conf_path(:ipv4)
+    dhclientv6_config_file = ipv6[:config_file] || Nerves.Network.DhclientConf.default_dhclient_conf_path(:ipv6)
 
     children = [
       supervisor(Registry, [:duplicate, Nerves.Dhclientv4], id: Nerves.Dhclientv4),
       supervisor(Registry, [:duplicate, Nerves.Dhclient], id: Nerves.Dhclient),
       worker(Nerves.Network.Resolvconf, [resolvconf_file, [name: Nerves.Network.Resolvconf]]),
-      worker(Nerves.Network.Dhclientv4Conf, [dhclientv4_config_file, [name: Nerves.Network.Dhclientv4Conf]]),
+      Supervisor.child_spec({Nerves.Network.DhclientConf, [dhclientv4_config_file, [name: Nerves.Network.DhclientConf.Ipv4]]}, id: Nerves.Network.DhclientConf.Ipv4),
+      Supervisor.child_spec({Nerves.Network.DhclientConf, [dhclientv6_config_file, [name: Nerves.Network.DhclientConf.Ipv6]]}, id: Nerves.Network.DhclientConf.Ipv6),
+      #worker(Nerves.Network.DhclientConf, [dhclientv4_config_file, [name: Nerves.Network.DhclientConf.Ipv4]]),
+      #worker(Nerves.Network.DhclientConf, [dhclientv6_config_file, [name: Nerves.Network.DhclientConf.Ipv6]]),
```

Comment:
If these aren't needed its best to delete

Reply:
done